### ISSUE TITLE
Are the DockerHub images out of date?

### ISSUE THREAD

**Issue by westurner** [splunk/docker-splunk#338]
When I go to https://hub.docker.com/publishers/splunk , it says:

```quote
Splunk Enterprise
By Splunk • Updated 9 months ago

Splunk Enterprise Free (for docker)
By Splunk • Updated a year ago
```

This is confusing to me as a user; would I be installing old versions of splunk, or do the images pull a new version of splunk every time they're started?

**Comment by westurner** [comment]
... There are various ways to build and release a new container image with CI systems for every release.

**Comment by westurner** [comment]
And now I've found: https://hub.docker.com/u/splunk and https://hub.docker.com/r/splunk/splunk/

- [ ] Maybe update the https://hub.docker.com/publishers/splunk  description with a link to the current containers?

**Comment by nwang92** [comment]
As far as I'm aware, we're only publishing to https://hub.docker.com/u/splunk and in particular, the `splunk/splunk` + `splunk/universalforwarder` images in that repository.

I'm not too familiar with https://hub.docker.com/publishers/splunk or why it says it was last updated 9mo ago (we recently published new images just last week). Is this a separate repository @mikedickey?

**Comment by NateLedet** [comment]
Hi @nwang92, not sure if this is a separate issue or related...

I followed the u/splunk link to https://hub.docker.com/r/splunk/splunk and it shows it was updated 11 hours ago. 

On my local Docker, I completely purged all containers, images, volumes, etc to get a totally fresh slate: `$ docker system prune -all`

Once that was completed, I followed the instructions on the splunk/splunk  page:
`$ docker pull splunk/splunk:latest`
`$ docker run -d -p 8000:8000 -e "SPLUNK_START_ARGS=--accept-license" -e "SPLUNK_PASSWORD=P@ssword123" --name splunk splunk/splunk:latest`

Once it spins up, I go to my browser, localhost:8000 and log in. 
I click Help > About and it shows I'm running version 7.3.4.2

Does `docker pull splunk/splunk:latest` not pull 8.x?  I see the tags and I know I can run `docker pull splunk/splunk:8.0.2.1` and turn up Splunk that way, but I guess I'm not understanding why "latest" isn't actually pulling the latest version of Splunk down. 

Thanks.

edit: `$ docker pull splunk/splunk:edge` does pull down 8.0.2.1

**Comment by nwang92** [comment]
Just to go back to the original issue - I'm still not sure what https://hub.docker.com/publishers/splunk is or how that's different, but I strongly advise using and/or sticking with https://hub.docker.com/r/splunk/splunk.

> I followed the u/splunk link to https://hub.docker.com/r/splunk/splunk and it shows it was updated 11 hours ago.

We publish splunk/splunk:edge on a nightly basis, so that's probably what you're seeing there. Dockerhub is just reporting when the repo was last updated, but not necessarily which tags were updated.

That `latest` tag pointing to 7.3.4.2 does seem like an error on our publication side, let me investigate and see if I can get it pointing to the correct image.

**Comment by alishamayor** [comment]
@NateLedet `latest` should point to 8.0.2.1 now. Was an error on my end when I published the 7.3.4.2 patch

**Comment by nwang92** [comment]
Going to close this since all of the above is resolved. I don't have access to https://hub.docker.com/publishers/splunk so not sure how to change things there, but hopefully if other people come across this, search will lead them to this issue and they'll know to use https://hub.docker.com/r/splunk/splunk :)
